login/resources/css/signin.css

```diff
@@ -1,118 +1,141 @@
 html,
 .login-page {
-  height: 100%;
+    height: 100%;
 }
 
 @font-face {
-  font-family: "Montserrat";
-  font-style: normal;
-  font-weight: 400;
-  src: url("../fonts/Montserrat-VariableFont_wght.ttf") format("truetype");  
+    font-family: "Montserrat";
+    font-style: normal;
+    font-variation-settings: 'wght' 400;
```

Comment:
TIL that `font-weight` does not apply to variable weight fonts, `font-variation-settings` is applied instead